OpenRA.Game/Traits/RevealsShroud.cs

```diff
@@ -30,10 +34,21 @@ public RevealsShroud(RevealsShroudInfo info)
 
 		public void Tick(Actor self)
 		{
-			if (cachedLocation != self.Location)
+			var ownerShroud = self.Owner.Shroud;
+			if (!ownerShroud.ShroudEnabled && !ownerShroud.FogEnabled)
+				return;
```

Comment:
If you have any allies for which fog/shroud is not disabled but yours is (e.g. enable cheats and have one player in a team disable fog/shroud but the other not) then they will not get the update to their sight.